src/js/pagination/paging-collection.js

```diff
@@ -98,6 +106,35 @@
             },
 
             /**
+             * Returns the totalPages of the collection based on
```

Comment:
Could you please change `totalPages` here to `total pages`? The idea is that whoever is reading this shouldn't have to know about the internals of how the `PagingCollection` works.